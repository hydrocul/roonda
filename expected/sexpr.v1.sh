#!/bin/sh

#################################################
# attachedfile.txt:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/attachedfile.txt

Hello Hello

END_OF_ROONDA_SOURCE_FILE
#################################################

echo Hello
echo Hello World
echo 'Hello   World!'
cat $ROONDA_TMP_PATH/attachedfile.txt
