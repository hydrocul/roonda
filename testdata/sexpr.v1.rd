sh v1
(echo Hello)
(echo Hello   World)
(echo "Hello   World!")

(cat attachedfile.txt)
(cat attachedfile-2.txt)
(cat attachedfile-3.txt)

<< attachedfile.txt

    Hello Hello

attachedfile.txt >>

<< attachedfile-2.txt

    Hello Hello 2

>> attachedfile-2.txt

<< attachedfile-3.txt \

    Hello Hello 3

>> attachedfile-3.txt

