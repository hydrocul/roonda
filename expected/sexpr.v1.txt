Hello
Hello World
Hello   World!

    Hello Hello


    Hello Hello 2


Hello Hello 3

